cpu/stm32_common/cpu_common.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_cortexm_common
+ * @{
+ *
+ * @file
+ * @brief       Shared CPU specific function for the STM32 CPU family
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include "periph_cpu_common.h"
+
+void periph_clk_en(uint8_t bus, uint32_t mask)
```

Comment:
not quite true, they both have APB1 and APB2...

And I did not inline them for a good reason: they would not be optimized to the set and clear operation, as the function is not called with constant values. So not inlining them saves ROM, and the run-time overhead for the function call does not hurt at this place.